src/Assignments/SimpleProblemPage.tsx

```diff
@@ -44,13 +45,27 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
     const [modalLoading, setModalLoading] = useState<boolean>(false);
     const [selectedProblemId, setSelectedProblemId] = useState<number | null>(null);
     const [loading, setLoading] = useState(true);
-    const [error, setError] = useState('');
+    const [alert, setAlert] = useAlertState();
```

Comment:
Since you changed this, you should consider using `useMUIAlertState` and use the Material UI version of Alert.

Reply:
The API is similar, but MUI's `severity` is the equivalent of BS's `variant`.